addons/event/views/event_views.xml

```diff
@@ -444,9 +444,9 @@
                         domain="[('activity_ids.date_deadline', '&gt;', context_today().strftime('%Y-%m-%d'))]"/>
                     <group expand="0" string="Group By">
                         <filter string="Responsible" name="responsible" context="{'group_by': 'user_id'}"/>
-                        <filter string="Template" name="event_type_id" context="{'group_by': 'event_type_id'}"/>
-                        <filter string="Stage" name="stage_id" context="{'group_by': 'stage_id'}"/>
-                        <filter string="Start Date" name="date_begin" domain="[]" context="{'group_by': 'date_begin'}"/>
+                        <filter string="Template" name="eventtype" context="{'group_by': 'event_type_id'}"/>
```

Comment:
Why not being more verbose?:
```suggestion
                        <filter string="Template" name="group_by_event_type_id" context="{'group_by': 'event_type_id'}"/>
```

Reply:
I found like this one was the common pattern around odoo code... well, let them say what they prefer, I don't have a strong opinion on that 🤷🏼‍♂️